README.md

```diff
@@ -142,10 +145,12 @@ Run locally via `make test-sanity`
 
 Integration tests are dependent on:
 - A flightctl instance the tests can hit
-  - Locally the easiest way is to run `make deploy` from the main flightctl repository
 - The flightctl_host var inside integration_config.yml set to the running flightctl api service
 
-Run locally via `make test-integration`
+The easiest way to run tests locally is to:
+- Run `make deploy` from the main flightctl repository
+- Run `make write-integration-config` from this repository to create the proper integration config from your running services
```

Comment:
question: is this something a user would use to manipulate an integration test? I guess the main question is can we just bundle this into `make test-integration`

Reply:
also in flightctl we use <type>-test I also see `ansible-test`

Reply:
Yeah thats a good point, i should just bundle them both.  And can rename to be more inline w/ the -test postfix